<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1552</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Job Application Tracker (Developer: Tianhao Zhang)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Title_JobTracker">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>701</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>64</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Job Application Tracker</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>1521</width>
      <height>711</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Company</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Job</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Weblink</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="AddJob">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>10</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add Job</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteJob">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>10</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Edit">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadData">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Load Data</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1552</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
